packages/core/lib/lockdown.umd.js

```diff
@@ -3027,7 +3027,7 @@ const tagError = (err, optErrorName = err.name) => {
  */
 const makeError = (
 optDetails = redactedDetails`Assert failed`,
-ErrorConstructor = globalThis.Error,
+ErrorConstructor = Error,
```

Comment:
You can't modify this. It's a vendored SES build

Reply:
yea i was asking about it [here](https://github.com/LavaMoat/LavaMoat/pull/360#discussion_r973700418)

what are my options then? can we change that at SES? any reason for having to use `globalThis.Error` instead of just `Error`?

Reply:
as agreed with @naugtur,

undo any changes made to this file 5ebd45b02593785fa8ffe6ed26124a1da7f905ea and avoid scuttling `Error` 785b4a71f

Reply:
ok